pybamm/models/submodels/electrode/base_electrode.py

```diff
@@ -151,8 +156,8 @@ def _get_standard_current_collector_potential_variables(self, phi_s_cn, phi_s_cp
             "Positive current collector potential [V]": phi_s_cp_dim,
             "Local voltage": V_cc,
             "Local voltage [V]": self.param.ocv_ref + V_cc * pot_scale,
-            "Terminal voltage": V,
-            "Terminal voltage [V]": V_dim,
+            "Terminal voltage": V - delta_phi_contact,
```

Comment:
why is this minus and not plus?

Reply:
also would be good to add contact resistance as its own variable

Reply:
"Contact resistance [Ohm]" is a parameter. Did you mean make "Contact overpotential [V]" a variable? I can do that if you think it's appropriate.

Reply:
Yes I meant contact overpotential as a variable. Also smart to ignore my dumb minus vs plus comment, brain fart on my part